--- a/qelectrotech.pro
+++ b/qelectrotech.pro
@@ -13,10 +13,10 @@
         QET_LANG_PATH              = 'share/qelectrotech/lang/'
         QET_EXAMPLES_PATH          = 'share/qelectrotech/examples/'
         QET_LICENSE_PATH           = 'doc/qelectrotech/'
-        QET_MIME_PACKAGE_PATH      = '../share/mime/packages/'
+        QET_MIME_PACKAGE_PATH      = 'share/mime/packages/'
         QET_DESKTOP_PATH           = 'share/applications/'
         QET_ICONS_PATH             = 'share/icons/hicolor/'
-        QET_MAN_PATH               = 'man/'
+        QET_MAN_PATH               = 'share/man/'
         QET_APPDATA_PATH           = 'share/appdata'
 }
 win32 {
